crates/bevy_window/src/lib.rs

```diff
@@ -33,6 +35,18 @@ impl Default for WindowPlugin {
     }
 }
 
+pub struct CursorPosition {
+    pub positions: HashMap<WindowId, Vec2>,
+}
+
+impl Default for CursorPosition {
+    fn default() -> Self {
+        Self {
+            positions: HashMap::default(),
+        }
+    }
+}
```

Comment:
I'm pretty sure we can simplify this to:
```rust
#[derive(Default)]
pub struct CursorPosition {
    pub positions: HashMap<WindowId, Vec2>,
}
```
and remove the explicit `Default` implementation.

Reply:
You're right! I changed it to that now. It didn't work for me originally, but perhaps that was when I using std collections HashMap implementation?

In any case whatever it was, now just the derive works. :+1: